test/unit/lib/metrics.js

```diff
@@ -208,6 +212,15 @@ describe('lib/metrics.js', () => {
               username: 'user'
             }));
           });
+          describe('non-blocked record', () => {
+            before(() => {
+              resetStubs();
+              register.resetMetrics();
+              subject('birth', 'user', 'my-client', ['group'], ['role'], startTime, finishTime, 1, false);
+            });
+            it('does not increment on lev.api.req.${dataSet}.blocked', () => hsIncrementStub.should.not.have.been.calledWith('lev.api.req.birth.blocked'));
```

Comment:
`callled()`? i.e. Do we need the 'with' stuff?

Reply:
Doesn't line 219 mean all the regular metrics will be recorded? And so the stub will have been called plenty!?

Reply:
Yes, you are right. Apologies.